pkg/migration/controllers/migration.go

```diff
@@ -39,6 +39,10 @@ const (
 	// StorkMigrationReplicasAnnotation is the annotation used to keep track of
 	// the number of replicas for an application when it was migrated
 	StorkMigrationReplicasAnnotation = "stork.libopenstorage.org/migrationReplicas"
+	// StorkMigrationSuspendAnnotation is the annotation used to enable/disable
+	// k8s object which requires field to be set to true/false
+	StorkMigrationSuspendAnnotation = "stork.libopenstorage.org/migrationSuspend"
```

Comment:
Why do we need to store that in the annotation? I think we can always suspend it on migration if `StartApplications` is false and trigger it when migrations are activated/deactivated